cypress/e2e/po/components/labeled-select.po.ts

```diff
@@ -69,4 +69,12 @@ export default class LabeledSelectPo extends ComponentPo {
   filterByName(name: string) {
     return this.self().type(name);
   }
+
+  static getGkeVersionSelect() {
```

Comment:
these are specific gke usages of the component and should be added to a gke specific po (labeled-select.po is generic)